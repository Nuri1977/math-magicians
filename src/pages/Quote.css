.quote-title {
  font-size: 1.8rem;
  line-height: 1.5;
  padding: 5rem 3rem;
}
