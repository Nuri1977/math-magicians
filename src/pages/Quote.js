import React from 'react';

export default function Quote() {
  return (
    <div>
      <h1>
        Mathematics is not about numbers, equations, computations, or algorithms:
        it is about understanding. -Qilliam Paul Th
      </h1>
    </div>
  );
}
