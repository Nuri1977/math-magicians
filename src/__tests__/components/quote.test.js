import renderer from 'react-test-renderer';
import Quote from '../../pages/Quote';

describe('Testing Quote component', () => {
  it('Quote renders correctly', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
