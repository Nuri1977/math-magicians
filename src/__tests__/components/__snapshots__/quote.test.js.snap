// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Quote component Quote renders correctly 1`] = `
<div
  className="container"
>
  <h1
    className="quote-title"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </h1>
</div>
`;
